robot/EDA/resources/AccountsAndContactsSettingsPageObject.py

```diff
@@ -0,0 +1,171 @@
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+from locators import eda_lex_locators
+from selenium.webdriver.common.keys import Keys
+
+
+@pageobject("Accounts_and_Contacts", "HEDA_Settings")
+class AccountsAndContactsSettingsPage(BasePage):
+
+    @property
+    def eda(self):
+        return self.builtin.get_library_instance('EDA')
```

Comment:
Do u also want to create a basepage to move this to that as u are already reorganizing stuff?

Reply:
Yes. That's the next WI I am picking up (W-041795). I need to create BaseEDAPage and also version the locator file

Reply:
I actually want to keep the PRs short and hence divided it into different WIs.
But creating the page objects in itself was rather a big change. So deferring the rest to subsequent PRs.